rpcs3/Gui/GLGSFrame.cpp

```diff
@@ -1,18 +1,41 @@
 #include "stdafx.h"
 #include "stdafx_gui.h"
 #include "GLGSFrame.h"
+#include "config.h"
 
 GLGSFrame::GLGSFrame() : GSFrame("OpenGL")
 {
-	const int context_attrs[] =
+	if (rpcs3::config.rsx.d3d12.debug_output.value())
 	{
-		WX_GL_RGBA,
-		WX_GL_DEPTH_SIZE, 16,
-		WX_GL_DOUBLEBUFFER,
-		0
-	};
+		const int context_attrs[] =
+		{
+			WX_GL_RGBA,
+			WX_GL_DEPTH_SIZE, 16,
+			WX_GL_DOUBLEBUFFER,
+			WX_GL_MAJOR_VERSION, 3,
+			WX_GL_MINOR_VERSION, 3,
+			WX_GL_CORE_PROFILE,
+			WX_GL_DEBUG,
```

Comment:
@Nekotekina That parameter does not take a value from looking at the sources. wxwidgets is inconsistent like that.